circle.yml

```diff
@@ -0,0 +1,35 @@
+machine:
+  node:
+    version: 0.10.40
+  environment:
+    LANG: en_US.utf8
+dependencies:
+  pre:
+    - npm -g install npm@latest-2
+    - echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+    - gem install mime-types -v 2.6.2
+    - gem install rest-client -v 1.8
+    - gem install crowdin-cli -v 0.5.5
+  override:
+    - npm install
+test:
+  override:
+    - npm run test
```

Comment:
Yeah. I'm not sure how CircleCI would react to me not giving it a command. Hence why I modified package.json to not `exit 0` anymore 😅 Also, I would like to write tests for it, despite how simplistic it is. No harm doing it really.